comscore/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "@theoplayer/comscore-connector-web",
+  "version": "1.0.21",
```

Comment:
Why are we starting at version 1.0.21?

Reply:
The last version in our private repo was 1.0.20

Reply:
But it was never published to the public npm registry? There's no [@theoplayer/comscore-connector-web](https://www.npmjs.com/package/@theoplayer/comscore-connector-web) package there yet.

That said, I'm okay with continuing the version numbering from the old connector. 👍